src/test/java/com/foundation/salesforce/steps/TaskAcceptanceSteps.java

```diff
@@ -37,77 +38,90 @@
     private ValidatableResponse json;
     private Response response;
     private TaskApi taskApi;
-
-    Task task = new Task();
+    private Task task;
 
     /**
-     * Retrieves an authentication token in order to be able to access to Salesforce platform.
+     * Class constructor.
```

Comment:
Please be a little more specific in this comment.

Reply:
OK, I'll be more specific in this section. Promise.

Reply:
Done.